src/components/Workflow/ScriptureTable.tsx

```diff
@@ -142,7 +142,25 @@ interface AudacityInfo {
   wf: IWorkflow;
   index: number;
 }
-
+/*
+function useTraceUpdate(props: any) {
+  const prev = useRef(props);
+  useEffect(() => {
+    const changedProps = Object.entries(props).reduce((ps, [k, v]) => {
+      if (prev.current[k as keyof typeof prev] !== v) {
+        (ps as any)[k] = [prev.current[k], v];
+      }
+      return ps;
+    }, {});
+    if (Object.keys(changedProps).length > 0) {
+      console.log('Changed props:');
+      Object.keys(changedProps).forEach((key) => {
+        console.log(key, changedProps[key as keyof typeof changedProps]);
+      });
+    }
+    prev.current = props;
+  });
+} */
```

Comment:
This doesn't seem to belong here. Does it belong in its own file under utils? You could probably comment out the contents of the function and then it would be ok with it there.

Reply:
It is commented out now...I'm not sure what you mean by comment out the contents of the function.  I've had to look this up multiple times...I could put it in as a util though.